test.sh

```diff
@@ -3,9 +3,11 @@
 
 set -eux
 
-HOST=http://localhost:8420
+HOST=${HOST:-http://localhost:8420}
```

Comment:
where does this `test.sh` run? i believe `$HOST` is a common envar that is often set by login shells and whatsuch. like, if i ran this on my MacBook, it would make `HOST=Andrew.K-MacBook-Pro` which is neither an HTTP address nor does it have a port -- so curl would think `http://Andrew.K-MacBook-Pro:80`

Reply:
good catch 👍  renamed the variable.